### ISSUE TITLE
TypeIntrinsics_il test build fails

### ISSUE THREAD

**Issue by BruceForstall** [dotnet/runtime#1265]
It looks like the build of the TypeIntrinsics_il test (added in #1157) fails:
```
TypeIntrinsics_il.il(20): error : syntax error at token '&' in:     ldtoken    [System.Runtime]System.String& [F:\gh\runtime\src\coreclr\tests\src\JIT\Intrinsics\TypeIntrinsics_il.ilproj]
```

The test was added as a pri-1 test, but I don't see evidence that the PR triggered Pri-1 (outerloop) testing (it hasn't been in good shape recently anyway).

**Comment by BruceForstall** [comment]
@EgorBo 
cc @jkotas @AndyAyersMS

**Comment by janvorli** [comment]
Also, the test compilation prints a warning:
TypeIntrinsics_il.il(20,0): warning : Reference to undeclared extern assembly 'System.Runtime'. Attempting autodetect

**Comment by EgorBo** [comment]
@janvorli fixed in #1271 thanks
